backend/api/Services/IsarService.cs

```diff
@@ -243,6 +245,45 @@ public async Task<IsarMission> StartLocalizationMission(Robot robot, Pose locali
             );
             return isarMission;
         }
+        public async Task<IsarMission> StartMoveArm(Robot robot, string armPosition)
+        {
+            string armPositionPath = $"schedule/move_arm/{armPosition}";
+            var response = await CallApi(
+                HttpMethod.Post,
+                robot.IsarUri,
+                armPositionPath
+            );
+
+            if (!response.IsSuccessStatusCode)
+            {
+                var (message, statusCode) = GetErrorDescriptionFoFailedIsarRequest(response);
+                string errorResponse = await response.Content.ReadAsStringAsync();
+                _logger.LogError("{message}: {error_response}", message, errorResponse);
+                throw new MissionException(message, statusCode);
```

Comment:
Implement same error message handling as commented earlier

Reply:
improved it